### PR TITLE
update to openzeppelin 1.12

### PR DESCRIPTION
**Pull request by ddbit**
update to openzeppelin 1.12, other minor changes in testcase formatting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ValorToken",
-  "version": "1.0.0",
+  "version": "0.1.1",
```
Comment (BrJu) [inline package.json]: Was it supposed to be version 1.0.1 ?
```suggestion
  "version": "1.0.1",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ValorToken",
-  "version": "1.0.0",
+  "version": "0.1.1",
```
Comment (BrJu) [inline package-lock.json]:
```suggestion
  "version": "1.0.1",
```

File: package.json [hunk 3]
```diff
@@ -26,7 +26,8 @@
     "chai-as-promised": "^7.1.1",
     "chai-bignumber": "^2.0.2",
     "chai-bn": "^0.1.1",
-    "openzeppelin-solidity": "1.11.0",
+    "bn.js": "^4.11.0",
```
Comment (BrJu) [inline package.json]: 4.11.8 ?

File: package.json (hunk 1 above)
Comment (ddbit) [reply package.json]: no we were on track for version 0.1.1 (a release candidate)

File: package-lock.json (hunk 2 above)
Comment (ddbit) [reply package-lock.json]: It's ok, we are not changing the interface of the contract so it's good to advance only the minor version, the previous value was there by mistake

**Comment by ddbit** [comment]
the version 0.1.1 is correct as we were on rc track to 0.1.1 with previous releases and no change to the ABI was done in this branch

**Comment by ddbit** [comment]
the version 0.1.1 is correct- we were on RC 0.1.1 with previous tag and no changes in ABI justifies a major version increment. The 1.0.0 was a previous mistake

**Pull request merged**